pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -22,6 +22,8 @@ import (
 	"strings"
 	"time"
 
+	"istio.io/istio/pilot/pkg/features"
+
```

Comment:
remove

Reply:
features package is used on the line "if features.SignCertAtK8sCa.Get() {"

Reply:
The blank line under "istio.io/istio/pilot/pkg/features" has been removed.